crates/bevy_input/src/gamepad.rs

```diff
@@ -154,6 +154,9 @@ impl ButtonSettings {
     }
 }
 
+/// Stores settings for axis.
+///
+/// The valid range is from -1.0 to 1.0, inclusive.
```

Comment:
I feel like this doesn't really communicate what the `AxisSettings` are used for. Maybe something about how it defines the sensitivity range and threshold for an axis?

Reply:
Can you have a look on the new documentation? I added more details

Reply:
Yeah I think your `Defines the sensitivity range and threshold for an axis.` is better. Thanks!